feat: Clean up settings UI

SuaYoo opened the pull request:
Some minor UI cleanup in preparation for new section for https://github.com/webrecorder/browsertrix/issues/1992
 
<!-- Fixes #issue_number -->

### Changes
- Renames "Org Settings" -> "Settings"
- Reduces gap between settings panel heading and panel
- Always show "Pending Invites" section and update heading styles to match panel heading
- Update "Current Plan" and "Usage History" sections to be on the same hierarchical level under "Billing"
- Refactors `<btrix-org>` to move `isAdmin` and `isCrawler` helpers to app state

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Org nav bar | <img width="669" alt="Screenshot 2024-08-13 at 6 08 10 PM" src="https://github.com/user-attachments/assets/30d8d52a-c576-4484-91e4-b3b2726d8bf4"> |
| Settings - Members | <img width="1056" alt="Screenshot 2024-08-13 at 5 58 17 PM" src="https://github.com/user-attachments/assets/632c3897-d9c0-4f86-8c50-0532c399c03e"> |
| Settings - Billing | <img width="1029" alt="Screenshot 2024-08-13 at 5 58 30 PM" src="https://github.com/user-attachments/assets/ba511546-c7fd-4104-a51e-bdbb7be967a0"> |

### Opinions

We didn't discuss this as a team, but I feel like the "Org" in "Org Settings" is redundant since they're settings that will always apply to the whole org, and the link to settings is in the org-level navigation bar.

I'm proposing the change in connection to https://github.com/webrecorder/browsertrix/issues/1992, since "Org Settings" might actually be slightly misleading as org information settings, rather than also being the place for org-wide crawling settings. I referenced a few other web apps and it's pretty consistently just "Settings", rather than "Team Settings" or "Organization Settings", etc.

SuaYoo committed:
move is admin

SuaYoo committed:
use isAdmin and isCrawler from state

SuaYoo committed:
fix gap

SuaYoo committed:
update sections

Shrinks99 commented:
"Settings" works for me, also has docs changes / implications!  Can change those if you'd like?

EDIT: Looks like we didn't add the billing tab in the last update either heh, I'm slacking!

SuaYoo commented:
> "Settings" works for me, also has docs changes / implications! Can change those if you'd like?

Good point, yeah if you want to commit doc changes to this PR!

Shrinks99 commented:
Upon further review, we now have "Account Settings" and "Settings".  I think that using "Org" as a prefix to deliate between the two is helpful.  That said, I agree that the "org" prefix is redundant in the menu.

Alternatively we could rename "Account Settings" to something else, "User Profile" perhaps?  I'll probably keep the page as "Organization Settings" (currently "Org Settings") in the docs?

SuaYoo committed:
add back org

SuaYoo commented:
> Upon further review, we now have "Account Settings" and "Settings". I think that using "Org" as a prefix to deliate between the two is helpful. That said, I agree that the "org" prefix is redundant in the menu.

Added back "Org" to page heading:
<img width="642" alt="Screenshot 2024-08-19 at 9 02 01 AM" src="https://github.com/user-attachments/assets/cb0e7e1e-8f3c-4cf3-a52a-8d8f7bf970fa">

And changed "Account Settings" icon to be distinct from org settings icon:
<img width="259" alt="Screenshot 2024-08-19 at 9 02 59 AM" src="https://github.com/user-attachments/assets/0a399488-38f5-49ea-b3e4-c97815be7dde">
